src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationILGen.cs

```diff
@@ -66,21 +66,6 @@ internal ModuleBuilder ModuleBuilder
         }
         internal TypeAttributes TypeAttributes { get { return _typeAttributes; } }
 
-        private static readonly Dictionary<string, Regex> s_regexs = new Dictionary<string, Regex>();
```

Comment:
I assume this was just dead code and wasn't used by reflection, can you confirm @krwq?

Reply:
It wasn't dead, it was used by the method below to cache the regexes being created (NewRegex).

Reply:
Oh nvm, I just saw where it was used and see why it was removed. disregard.

Reply:
I'm not sure cc: @StephenMolloy @HongGit

Reply:
It's dead now. ;) Looks fine to me.